samples/client-sdk/knocking/index.js

```diff
@@ -331,18 +343,7 @@ const createOwnerCall = async ({ name, url, token }) => {
 
   // Let owner join the meeting
   try {
-    const join = await callObject.join({ userName: name, url, token });
-
-    // Confirm the participant is an owner of the call (i.e. can respond to knocking)
-    if (join.local.owner !== true) {
-      console.error('This participant is not a meeting owner!');
-    } else {
-      console.log('This participant is a meeting owner! :)');
-    }
-
-    // Update UI after call is joined
-    hideLoadingText('owner');
-    showOwnerPanel();
+    await callObject.join({ userName: name, url, token });
```

Comment:
Nit: since we do not use the awaited value anymore, it _might_ be nicer to add a promise catch call instead of awaiting in a try-catch. I.e., `callObject.join().catch((error) => ....`

WDYT?